Orthtree/include/CGAL/Orthtree_traits_face_graph.h

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2023  INRIA (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Sebastien LORIOT
+
+
+#ifndef CGAL_ORTHREE_TRAITS_FACE_GRAPH_H
+#define CGAL_ORTHREE_TRAITS_FACE_GRAPH_H
+
+#include <CGAL/license/Orthtree.h>
+
+#include <CGAL/Orthtree_traits_base_for_dimension.h>
+
+#include <boost/graph/graph_traits.hpp>
+#include <CGAL/Dimension.h>
+#include <CGAL/Bbox_3.h>
+
+namespace CGAL {
+
+/*!
+  \ingroup PkgOrthtreeTraits
+
+  The class `Orthtree_traits_face_graph` can be used as a template parameter of
```

Comment:
Where will faces end up in the tree?  In each node where the bbox intersects the face?  Is that evident, or must that be documented.

Reply:
Doc is now more specific.